concordium-smart-contract-testing/Cargo.lock

```diff
@@ -0,0 +1,1705 @@
+# This file is automatically @generated by Cargo.
```

Comment:
What is this here for?

Reply:
This project was removed from this repository.

Reply:
ups